### PR TITLE
feat(query/stdlib): unskip agg window median test

### LINKED ISSUE #19565: Flux end-to-end test failing with "field type conflict"
The Flux end-to-end test `chronograf/aggregate_window_median_test.flux` is failing post-split with Flux upgraded to v0.83.1, with this error:
```
   --- FAIL: TestFluxEndToEnd/testing/chronograf/aggregate_window_median (0.14s)
            end_to_end_test.go:226: error calling function "inspect" @0:0-0:0: error calling function "case" @testing.flux|20:10-20:16: error calling function "loadStorage" @aggregate_window_median_test.flux|105:11-105:43: error calling function "chain" @9:9-12:3: field type conflict
```
This error message is not present in the `flux` repo and seems to be coming from storage.

The test will be skipped in `influxdb/query/stdlib/testing/testing.go`.

### PR DESCRIPTION
**Pull request by fchikwekwe**
Closes #19565

This change required a patch release of flux, so it bumps up the flux version from v0.83.1 to v0.83.2. 

It unskips a test that was previously failing in influxDB CI. The fix was to [correct the test's input data](https://github.com/influxdata/flux/pull/3233) to have a consistent `_value` type.

### CODE REVIEW

**Review by nathanielc — APPROVED** [review]
Thanks for staying on top of these tests and getting them working.

**Comment by fchikwekwe** [comment]
If its alright, I'm gonna wait until Monday morning to merge this version out of caution.

**Pull request merged**
